.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,45 @@
+<!-- HTML Comments can be left in place or removed, dealers choice. They are present simply to guide you on your pull-request journey. --> 
```

Comment:
Question for anyone putting in PRs: Are HTML comments in the template a good idea?

Reply:
I'm just a lowly community member, but imo the comments make more sense to me for issues where people might need to be guided through filling out the template. Hopefully anyone putting in a PR can figure it out on their own?

Reply:
You would hope so, but for some where english is a second language or translated the comments may not be direct enough. I have also mistakenly entered information in a comment only to find out later it did not render correctly.

I like the approach longhorn has taken with issue templates a little more where it should be pretty obvious what needs to be filled in (https://raw.githubusercontent.com/longhorn/longhorn/master/.github/ISSUE_TEMPLATE/bug_report.md)

Ultimately I am ok with either approach tho, just wanted to open it to discussion.

Reply:
Would it make sense to do a trial run of both for a period of time, measure their success or failures, and go with the one that makes things best, at the end of that trial period?